### ISSUE TITLE
sensor.launch changes the lidar operating state from STANDBY to NORMAL

### ISSUE THREAD

**Issue by marcelop3** [ouster-lidar/ouster-ros#74]
Hi,

When I launch the sensor.launch the lidar immediately changes its operating state from STANDBY to NORMAL. 
Apparently this behaviour is due to a hardcode parameter in the following file/line:

https://github.com/ouster-lidar/ouster-ros/blob/bf8636552a775d2739b189da8318d875105545ad/src/os_sensor_nodelet.cpp#L284

I am aware that it is possible to change the operating state back to STANDBY through the `rosservice call /ouster/set_config`, but maybe it would be better to receive the config.operating_mode as a parameter, similar to what happens to the lidar_mode

https://github.com/ouster-lidar/ouster-ros/blob/bf8636552a775d2739b189da8318d875105545ad/src/os_sensor_nodelet.cpp#L198

https://github.com/ouster-lidar/ouster-ros/blob/bf8636552a775d2739b189da8318d875105545ad/src/os_sensor_nodelet.cpp#L285

It would give us flexibility to define when the sensor would really start spinning/getting data.

Thanks!

**Comment by hodnajit** [comment]
Good catch, I would like to set the operating_mode in ouster_ros driver for ROS2 too :)

**Comment by Samahu** [comment]
Definitely makes sense,

@hodnajit with regard to the ros2 driver it implements the ros2 lifecycle managment. which gives you a bit of control over when the node performs configuration and starts stream. I don't currently change operation mode from standby to normal when the node transitions from configure mode to active mode but that would be the goal (instead of adding a new launch file argument).

**Comment by Grisly00** [comment]
One problem I just noticed in the ROS2 implementation is, that if I send a `set_config` service call to the driver to switch to STANDBY mode, the sensor gets reinitialized and instantly switches back to NORMAL mode.
I am using the `sensor.independent.launch.xml` launch file.

**Comment by Samahu** [comment]
I don't think I currently handle the case if the user passes a config with  STANDBY to the `set_config` service :disappointed: . I will try to address this in a later update.
